File: #1295.find-numbers-with-even-number-of-digits.cpp
class Solution {
public:
    int findNumbers(vector<int>& nums) {

        int even = 0;
        int result = 0;
        for (int num : nums) {

            even = 0;
            while (num) {
                num /= 10;
                even++;
            }
            
            if (even % 2 == 0) {
                result++;
            }
        }
        return result;
    }
};

File: #1295.find-numbers-with-even-number-of-digits.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findNumbers = function(nums) {
    let count = 0;
    for (let num of nums) {
        let str = num.toString();
        if (str.length % 2 === 0) {
            count++;
        }
    }
    return count;
};
